### ISSUE TITLE
[Shaders] DOF doesn't work when motion blur is enabled

### ISSUE THREAD

**Issue by Jkmcameron** [sp614x/optifine#3963]
## Description of Issue
I've tried multiple different shader packs (SUES, Sildurs, etc) and in it seems that regardless of the shader pack being used, enabling motion blur causes DOF to no longer take effect

## Steps to Reproduce
1. Enable DOF
2. Observe that it works
3. Enable motion blur
4. Observe that DOF no longer works

## OptiFine Version
OptiFine 1.15.2 HD U G1 pre17

## Installation Method
I dropped the JAR file into my forge mods fodler

## Fabric/Forge Version
forge-31.1.77

## Other Installed Mods

AppleSkin
Better Sprinting
Bookshelf
Giselbaer's Durability Viewer
Hwyla
Just Enough Items (JEI)
Mouse Tweaks
No Recipe Book
Shulker Tooltip
Sound Filters
SwingThroughGrass
Wawla - What Are We Looking At
Xaero's Minimap
Xaero's World Map

## F3 Debug Screenshot
![2020-05-17_13 03 28](https://user-images.githubusercontent.com/14365419/82154909-0dcb2480-983f-11ea-9aab-4d6a099ec101.png)

To open this menu, simply press `F3` on your keyboard while in-game. Please ensure the text is easily readable. Depending on the size/resolution of your monitor, you may have to adjust your GUI scale for all text to be completely visible on-screen.

## Prior Testing
If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
   Yes
- Does this happen in OptiFine standalone?
   Yes

**Comment by yezhiyi9670** [comment]
Isn't it intended? Most shaders has a tooltip indicating that DOF/MotionBlur will not work with MotionBlur/DOF.

**Comment by Jkmcameron** [comment]
> Isn't it intended? Most shaders has a tooltip indicating that DOF/MotionBlur will not work with MotionBlur/DOF.

no, trust me I checked, I don't see anything anywhere about DOF and motion blur being incompatible, I even looked it up. Can't imagine why those two things would be incompatible in this first place. Besides, the shaders allow you to enable both, if they can't be used together it'd been easy to not allow them both to be on at the same time

**Comment by sp614x** [comment]
Does DOF and Motion Blur work together in 1.14.4?

**Comment by rre36** [comment]
It says on the Motionblur tooltip in Sildurs that it is not compatible with Depth of Field. Latest SEUS however does not even seem to have Depth of Field.

**Comment by Jkmcameron** [comment]
> It says on the Motionblur tooltip in Sildurs that it is not compatible with Depth of Field. Latest SEUS however does not even seem to have Depth of Field.

Ah, I'd checked the tool tip for DOF and saw no mention of motion blur, but I didn't check motion blur to see if it mentioned DOF, seems like an oversight on their part. Since this isn't an issue with optifine then this thread can be closed
